diff --git a/node_modules/expo-age-range/ios/ExpoAgeRange.podspec b/node_modules/expo-age-range/ios/ExpoAgeRange.podspec
index 7883f2b..63e3e9a 100644
--- a/node_modules/expo-age-range/ios/ExpoAgeRange.podspec
+++ b/node_modules/expo-age-range/ios/ExpoAgeRange.podspec
@@ -14,7 +14,7 @@ Pod::Spec.new do |s|
     :ios => '15.1',
     :tvos => '15.1'
   }
-  s.swift_version  = '6.0'
+  s.swift_version  = '5.0'
   s.source         = { git: 'https://github.com/expo/expo.git' }
   s.static_framework = true
 
